src/hotspot/share/classfile/stackMapTableFormat.hpp

```diff
@@ -41,11 +41,11 @@ class verification_type_info {
   address tag_addr() const { return (address)this; }
   address cpool_index_addr() const { return tag_addr() + sizeof(u1); }
   address bci_addr() const { return cpool_index_addr(); }
+  NONCOPYABLE(verification_type_info);
 
  protected:
   // No constructors  - should be 'private', but GCC issues a warning if it is
```

Comment:
Have we checked this is still the case? Might be a very old gcc issue.

Reply:
I did a little bit of experimenting and couldn't get gcc to warn, though other compilers might do so.  I think such a warning is not unreasonable.

Reply:
Based on Kim's comment, I decided to leave the change as it currently is.